example_csv_for_batch_update_users.csv

```diff
@@ -0,0 +1,3 @@
+Email,New Email

```

Comment:
Minor: It would be better to place this in an `examples` folder, just to differentiate the scripts and the samples

Reply:
Agreed. Feel free to do that otherwise let me know if I should

Reply:
I can do it :)